includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1769,6 +1769,7 @@ public function get_metadata_from_order( $order ) {
 			'order_key'      => $order->get_order_key(),
 			'payment_type'   => $payment_type,
 			'signature'      => $this->get_order_signature( $order ),
+			'tax_amount'     => WC_Stripe_Helper::get_stripe_amount( $order->get_total_tax(), strtolower( $order->get_currency() ) ),
```

Comment:
I've considered naming it `total_tax`, but since we use `shipping_amount`, I opted for `tax_amount` here.